rope/contrib/fixsyntax.py

```diff
@@ -62,7 +62,7 @@ def old_pyname():
         new_code = pymodule.source_code
 
         def new_pyname():
-            newoffset = self.commenter.transfered_offset(offset)
+            newoffset = self.commenter.transferred_offset(offset)
```

Comment:
```suggestion
            newoffset = self.commenter._transferred_offset(offset)
```

This probably shouldn't be a public method.